definitions/npm/mysql_v2.x.x/test_mysql_v2.x.x.js

```diff
@@ -1,139 +1,143 @@
 // @flow
 
-import mysql from 'mysql';
-import type { Connection, Pool } from 'mysql';
+import mysql from "mysql";
+import type { Connection, Pool } from "mysql";
 
 // Connection
 
 let connection: Connection = mysql.createConnection({
-  host: 'localhost',
+  host: "localhost",
   port: 1234,
-  user: 'test_user',
-  password: '1234'
+  user: "test_user",
+  password: "1234"
 });
 
-connection = mysql.createConnection('mysql://root:1111@localhost/test');
+connection = mysql.createConnection("mysql://root:1111@localhost/test");
 
 connection.connect();
-connection.connect((err) => {});
+connection.connect(err => {});
 
 connection.end();
 connection.end(() => {});
 connection.end((error: ?Error) => {});
 
 connection.destroy();
 
-connection.query('SELECT 1');
-connection.query('SELECT * FROM `table` WHERE id = ?', [1]);
-connection.query({sql: 'SELECT * FROM `table` WHERE id = ?', nestTables: true});
-connection.query('', [1,'d',{a:1},[{b:1}]]);
+connection.query("SELECT 1");
+connection.query("SELECT * FROM `table` WHERE id = ?", [1]);
+connection.query({
+  sql: "SELECT * FROM `table` WHERE id = ?",
+  nestTables: true
+});
+connection.query("", [1, "d", { a: 1 }, [{ b: 1 }]]);
+
+let post = { id: 1, title: "Hello MySQL" };
+connection.query("INSERT INTO posts SET ?", post);
```

Comment:
The above _two lines_ were the only ones I added in the test file, the rest is `prettier` auto-formatting from the commit hook.